README.md

```diff
@@ -124,7 +124,32 @@ git clone git@github.com:catalyst/moodle-tool_excimer.git admin/tool/excimer
 ```
 
 Then login as admin (it should detect the extension), and click through the upgrade process.
+## Applying core patches
 
+For Moodle versions lower than 4.1 this plugin requires Moodle tracker MDL-75014 to be backported to maintain the plugin functionality.
+#### Moodle 3.5 - 4.0:
+Apply the patch:
+<pre>
+git am --whitespace=nowarn < admin/tool/excimer/patch/MOODLE_35_STABLE.diff
+</pre>
+The patch was created with following commands:
+<pre>
+// Starting on core moodle
+git checkout MOODLE_310_STABLE
+
+// Checkout new branch
+git checkout -b newbranch
+
+// Cherry-pick MDL-75014
+git cherry-pick 519c932cd63749843dbbb2af883793d5c937ee0c
+
+// Resolve any merge conflicts only accepting changes for the tracker (changes to upgrade.txt were omitted for simplicity when backporting) 
+git cherry-pick --continue
+
+// Create the patch
+git format-patch MOODLE_310_STABLE --stdout > admin/tool/excimer/patch/MOODLE_35_STABLE.diff
```

Comment:
This is good but I don't believe this is required here. It might be useful in the wiki / reusable workflows repository if not already.